README.md

```diff
@@ -40,44 +43,67 @@ function intro(person, line) {
-via `this`. That's what we're doing with the function `introWithContext` as
-invoked with either `call` or `apply`.
+Your solution probably has a similar repetition.
+
+What if we told JavaScript that instead of the record being a _parameter_, it
+could be assumed as a _context_ and thus accessible via `this`. To accomplish
+this, we can use either `call` or `apply`:
 
-The `introWithContext` function expects only a catchphrase as an argument. Both `call` and
-`apply` take a `thisArg` argument as their first argument (see their
-documentation for further clarification): that argument becomes the `this`
-_inside_ the function.  In the case of `call`, anything after the `thisArg`
-gets passed to the function like arguments inside of a `()`. In the case of
-`apply`, the contents in the `Array` get destructured and passed to the
-function like arguments inside of a `()`.
+```js
+function introWithContext(line){
+  return `${this.firstName} ${this.familyName} says: ${line}`
+}
+
+introWithContext.call(asgardianBrothers[0], phrase)
+//=> Thor Odinsson says: I like this brown drink very much, bring me another!
+
+introWithContext.apply(asgardianBrothers[0], [phrase])
+//=> Thor Odinsson says: I like this brown drink very much, bring me another!
+```
+
+Note that, unlike the `intro` function, `introWithContext` expects only a
+catchphrase as an argument. We can then call either `call` or `apply` on
+`introWithContext`, passing a `thisArg` object as the first argument; that
+argument becomes the `this` _inside_ the function. (See the documentation for
+`call` and `apply` for further clarification.)
+
+As seen above, the function calls using `call` or `apply` are nearly identical:
+the only difference is in how additional arguments (if any) are passed to them.
+In the case of `call`, which can take any number of arguments, the additional
+arguments are listed individually after the `thisArg`. They are passed along to
+the function that `call` is being called on the same way that arguments inside
+of a `()` are passed. Additional arguments to `apply`, on the other hand, are
+passed inside an array. When `apply` is called, the array of arguments gets
+destructured and the arguments it contains are passed along just as they are
+with `call`.
+
+All three approaches for calling our introduction function yield the same
+results:
+
+```js
+intro(asgardianBrothers[0], phrase) === introWithContext.call(asgardianBrothers[0], phrase) //=> true
+intro(asgardianBrothers[0], phrase) === introWithContext.apply(asgardianBrothers[0], [phrase]) //=> true
+
+const complaint = "I was falling for thirty minutes!"
+intro(asgardianBrothers[1], complaint) === introWithContext.call(asgardianBrothers[1], complaint) //=> true
+intro(asgardianBrothers[1], complaint) === introWithContext.apply(asgardianBrothers[1], [complaint]) //=> true
+```
 
 > **ES6 ALERT**: Some might wonder: if we have destructuring of `Array`s, why
```

Comment:
Unrelated to your edits, but this is also incorrect/confusing - destructuring doesn't make sense here. Destructuring is for variable assignment, so how would you destructure when calling a method? Maybe the author meant the spread operator, since these examples are equivalent:

```js
const args = [1,2,3]
function callMe(num1, num2, num3) {
  console.log(this, num1, num2, num3)
}

callMe.apply({ hi: "there" }, args) // => { hi: "there" }, 1, 2, 3
callMe.call({ hi: "there" }, ...args) // => { hi: "there" }, 1, 2, 3
```

I would just remove the aside since this content is already pretty tough stuff.

Reply:
Ah, ok - missed that. I'll take it out